src/app/app.component.ts

```diff
@@ -77,6 +81,37 @@ export class AppComponent implements OnDestroy {
 	readonly autocompleteOptions$!: Observable<string[]>;
 
 	constructor() {
+		/*
+        Problem:
+        dateRangeFormGroup.controls.dateRangeFormControl emits value even if emitEvent == false
```

Comment:
Is this comment still relevant? If so, let's reword, it's a bit confusing on what it's saying

Also formatting is really weird (white space is all over the place)